libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -413,6 +413,30 @@ float AP_MotorsHeli_Dual::get_swashplate (int8_t swash_num, int8_t swash_axis, f
                 swash_tilt = -0.45f * _dcp_scaler * roll_input + coll_input;
             }
         }
+    }
+    else if (_dual_mode == AP_MOTORS_HELI_DUAL_MODE_INTERMESHING) {
```

Comment:
a tiny formatting thing - we should put the "else" on the line above, after the curly bracket.

Reply:
Do I fix this in a new commit or should I amend last commit?

Reply:
Pitt, you can do a soft reset using this command
git reset HEAD-1
Make the requested changes. Then add and commit.  Then you will do a force push to your remote by adding “-f” after the push command like this
git push origin Synchropter -f
That will update the PR too

Reply:
Fixed now